pitest-maven-verification/pom.xml

```diff
@@ -32,13 +32,6 @@
 			<plugin>
 				<artifactId>maven-failsafe-plugin</artifactId>
 				<version>2.12</version>
-				<dependencies>
-					<dependency>
-						<groupId>org.apache.maven.surefire</groupId>
-						<artifactId>surefire-junit47</artifactId>
-						<version>2.12</version>
```

Comment:
I've just found the previous commit "try to fix hanging build for jdk7". Did it help?